tests/test_de.rs

```diff
@@ -20,6 +20,7 @@ where
     assert_eq!(*expected, deserialized);
 
     let value: Value = serde_yaml_ng::from_str(yaml).unwrap();
+    println!("{:?}", value);
```

Comment:
It is better to use `dbg!` macro to make visible values in failed tests

Reply:
thank you. I will correct it at weekends.